spec.html

```diff
@@ -27139,7 +27139,7 @@
     <!-- es6num="21.1.3" -->
     <emu-clause id="sec-properties-of-the-string-prototype-object">
       <h1>Properties of the String Prototype Object</h1>
-      <p>The String prototype object is the intrinsic object <dfn>%StringPrototype%</dfn>. The String prototype object is an ordinary object. The String prototype is itself a String object; it has a [[StringData]] internal slot with the value *""*.</p>
+      <p>The String prototype object is the intrinsic object <dfn>%StringPrototype%</dfn>. The String prototype object is an exotic String object and has the internal methods specified for such objects. It has a [[StringData]] internal slot with the value *""*.</p>
```

Comment:
An object is only exotic if it uses a different implementation for any of the internal methods. And since String.prototype is a String object instance, and String objects are exotic String objects, String.prototype is also exotic.